src/test/java/org/dbunit/dataset/builder/DataSetBuilderIntegrationTest.java

```diff
@@ -62,6 +62,7 @@ public void testBuild() throws Exception {
 
         ReplacementDataSet expected = new ReplacementDataSet(new FlatXmlDataSetBuilder().build(
                 this.getClass().getResourceAsStream("/reference.xml")));
+        expected.addReplacementObject("[EPOCH_TIME]", new Date(0));
```

Comment:
Sorry, when I said it would not work if TimeZone is not set to UTC/GMT, I meant if for Timezone in "GMT-N".
GMT+2 works because it doesn't change the date in new Date(0), but using GMT-1, for example, changes the date to "1969-12-31", then you would see red bar, as reference.xml declares expectation as "1970-01-01".

Could you please add this line `TimeZone.setDefault(TimeZone.getTimeZone("GMT-1"));` to the test and then try running it again?